src/spring-cloud/azext_spring_cloud/_validators.py

```diff
@@ -180,6 +181,11 @@ def validate_vnet(cmd, namespace):
         raise CLIError('--app-subnet and --service-runtime-subnet should not be the same.')
 
     vnet_obj = _get_vnet(cmd, vnet_id)
+    instance_location = namespace.location

```

Comment:
will there be format difference? like `west us` vs `westus`. :)

Reply:
Fixed.